### ISSUE TITLE
About the training

### ISSUE THREAD

**Issue by mawei-north** [Tombs98/CAPTNet#4]
Hello, I'm intrigued by your work! Are you putting all the task dataset images in the same folder during training? How was data equalization implemented in your paper?

**Comment by Tombs98** [comment]
> Hello, I'm intrigued by your work! Are you putting all the task dataset images in the same folder during training? How was data equalization implemented in your paper?

Yes, in the same folder. In order to maintain the approximate amount of data for each degradation type, we use data enhancement methods such as flipping and so on.
